drivers/wifi/nrf700x/Kconfig

```diff
@@ -74,10 +74,23 @@ config NRF_WIFI_IF_AUTO_START
 
 config NRF_WIFI_PATCHES_BUILTIN
 	bool "Store nRF700x FW patches as part of the driver"
+	depends on !NRF_WIFI_PATCHES_EXT_FLASH_STORE
 	help
 	  Select this option to store nRF700x FW patches as part of the driver.
 	  This option impacts the flash memory size of the application.
-	default y
+	default y if !NRF_WIFI_PATCHES_EXT_FLASH_STORE
+
+choice NRF_WIFI_PATCHES_EXT_FLASH_SUPPORT
```

Comment:
I guess you could have `NRF_WIFI_PATCH_CHOICE` choice that would contain `_BUILTIN`, `_EXT_FLASH_STORE` and `_EXT_FLASH_XIP` to get rid off `NRF_WIFI_PATCHES_EXT_FLASH_DISABLED` that duplicates `NRF_WIFI_PATCHES_BUILTIN`.

Reply:
No, `BUILTIN` is needed for XIP, so, cannot be part of the choice.